doc/developer-guides/hld/hld-APL_GVT-g.rst

```diff
@@ -131,7 +131,7 @@ The architecture of API forwarding is shown in :numref:`api-forwarding`:
    API Forwarding
 
 A frontend driver is employed to forward high-level API calls (OpenGL,
-Directx, and so on) inside a VM, to a Backend driver in the Hypervisor
+Directx, and so on) inside a VM to a Backend driver in the Hypervisor
```

Comment:
"Backend" is spelled with a capital "B" while "frontend" (earlier) does not use a capital "f". Should we unify this?

Reply:
Agreed; I'll make them all lowercase.